settings.gradle

```diff
@@ -1,2 +1,4 @@
 include ':file_operations'
 include ':app', ':commons_compress_7z'
+
+enableFeaturePreview("TYPESAFE_PROJECT_ACCESSORS")
```

Comment:
please explain this part

Reply:
https://docs.gradle.org/7.0/userguide/declaring_dependencies.html#sec:type-safe-project-accessors